assembly/std/index.ts

```diff
@@ -32,8 +32,10 @@ export function call(
   functionName: string,
   args: Args,
   coins: u64,
-): string {
-  return env.call(at.toByteString(), functionName, args.serialize(), coins);
+): StaticArray<u8> {
+  // IMPORTANT TODO: change this after `Args` update
```

Comment:
Args feature is now ok no?

Reply:
Yes but it does not serialize into the right data type, need to sync with the inno team on that to see if they want me to take care of it or if they'd rather do it themselves

Reply:
Done.